docs/.vitepress/theme/index.ts

```diff
@@ -11,6 +12,9 @@ export default {
     })
   },
   enhanceApp({ app, router, siteData }) {
-    // ...
+    app.config.globalProperties.$formatDate = (date: string) => {
+      return DateTime.fromISO(date)
+        .toLocaleString(DateTime.DATE_HUGE, { locale: 'en-US' })
+    }
```

Comment:
I created this small helper with Luxon to format the post dates from ISO to a standard format, so we won't have to do it manually.